Tools/Replay/MsgHandler.h

```diff
@@ -115,6 +115,7 @@ inline void MsgHandler::field_value_for_type_at_offset(uint8_t *msg,
 {
     /* we register the types - add_field_type - so can we do without
      * this switch statement somehow? */
+
```

Comment:
remove this one ?

Reply:
@khancyr I'm not sure this entire patch set is tenable. It works great with GCC8.1. GCC will absolutely barf if the older compiler doesn't understand the warning you are trying to turn off. If the compiler had been written to be more forward compatible it would be fine... But it's very far from that.

Reply:
ho! this whitespace addition is a warning fix ? I wasn't aware, I was just thinking that it was a remaining whitespace from previous try